spec/monkey_patches.rb

```diff
@@ -65,18 +65,10 @@ def disable_all!
   end
 end
 
-if defined?(::Typhoeus.before)
```

Comment:
Ah, _this_ conditional is perhaps needed: we have some `Gemfile` variants where the Typheous gem is not defined, I believe.